README.md

```diff
@@ -4,10 +4,32 @@
 ### The keyboard lights in action
 ![](https://raw.githubusercontent.com/JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module/main/keyboard.webp)
 
-### A look at my front-end application
+### A look at the front-end application
 ![](https://raw.githubusercontent.com/ample-samples/web-portfolio/190384147cc67410606e54d72615f6b22b3a66ed/src/files/pictures/RGB%20Acer%20GUI.png)
 
-This application was created to help users control their keyboard lights in a user-friendly way. This is a Linux only application designed for users of Arch Linux. I created this to solve the problem I had which was when I had just started using Linux and found that there were no programs to control my laptop's lights in an easy way.
+This application was created to help users control their keyboard lights in a user-friendly way. This is a Linux only application designed for users of Arch Linux. This was created to solve the problem there being no programs to control acer keyboards on Linux.
+
+## Installation
+
+### Dependencies
+The GUI requires Python, Tkinter and CustomTkinter to run.
+
+#### Arch
+```
+sudo pacman -S python tk
+yay -S customtkinter
+```
+
+### Install
+```
+git clone https://github.com/DJPretzel-bit64/acer-predator-rgb-gui --recursive
+cd acer-predator-rgb-gui/acer-predator-turbo-and-rgb-keyboard-linux-module
+chmod +x ./*.sh
+sudo ./install_service.sh
+```
+
+### Run
+Open the "rgb Keyboard GUI" folder and run "RGB Control.py". To activate the custom profile, run 
```

Comment:
This line looks to be incomplete, is this intentional?

Reply:
Oh, yeah. I was writing that out and got distracted. I'll fix that.